.github/workflows/nightly.yml

```diff
@@ -108,6 +108,14 @@ jobs:
             -d zend_extension=opcache.so
             -d opcache.enable_cli=1
             -d opcache.jit_buffer_size=16M
+            -d opcache.jit_prof_threshold=0.000000001
+            -d opcache.jit_max_root_traces=10000000
+            -d opcache.jit_max_side_traces=10000000
+            -d opcache.jit_max_exit_counters=1000000
+            -d opcache.jit_hot_loop=1
+            -d opcache.jit_hot_func=1
+            -d opcache.jit_hot_return=1
+            -d opcache.jit_hot_side_exit=1
```

Comment:
These are probably not the best settings, because they will trigger JIT compilation on the first function entry or loop iteration and this sometime may lead to very unnatural traces that will never executed.

Anyway, I think it make sense to start with these settings and then change them to a bit less aggressive. (e.g. 2 or 3). 

Note, that these settings may increase the tests time.

Reply:
I am aware they increase test times, JIT compilation on first function entry/loop iteration is precisely why I chose those values, any other value already breaks reproduction of bugs like https://github.com/php/php-src/issues/12249